apps/daffio/src/app/design/device/laptop/laptop.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'daffio-laptop',
+  templateUrl: './laptop.component.html',
+  styleUrls: ['./laptop.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class LaptopComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
can remove the ngOnInit here and on line 9

Reply:
Resolved in a457728